Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -24,14 +24,17 @@
  *  Rev B  2 JUN 2017
  *
  *  Converted to Arduino pin numbering
+ *  at90usb1286 to arduino pin mapping is here:
+ *     https://labitat.dk/wiki/Panelolu_and_Printrboard_the_easy_way
```

Comment:
Correct. My comment is mainly due to the link being outdated and containing information that no longer pertains to Marlin. We have a website now, so if someone just throws together an update with Markdown, we may want to document this at marlinfw.org.